x/qgb/README.md

```diff
@@ -0,0 +1,139 @@
+# Quantum gravity bridge state machine
+
+This module contains the [Quantum gravity bridge](https://blog.celestia.org/celestiums/) state machine implementation.
+
+## State machine
+
+The QGB state machine handles the creation of the attestations requests:
+
+https://github.com/celestiaorg/celestia-app/blob/801a0d412631989ce97748badbd7bb676982db16/x/qgb/types/attestation.go#L15-L23
```

Comment:
[Question for reviewers]
Currently, markdown linter complains about bare URLs:
```
MD034/no-bare-urls Bare URL used [Context: "https://github.com/celestiaorg..."]
```
However, I guess in the case where we're providing snippets links, it should be okay. Since, using links to code makes it easier to scroll, syntax highlighting and navigation.

Do you think we should remove the rule? or I switch to copy-paste?

Reply:
I think we should keep the linter rule and provide a link name like so:

```markdown
The QGB state machine handles the creation of [`AttestationRequestI`](https://github.com/celestiaorg/celestia-app/blob/801a0d412631989ce97748badbd7bb676982db16/x/qgb/types/attestation.go#L15-L23) which can be a [`DataCommitment`](https://github.com/celestiaorg/celestia-app/blob/801a0d412631989ce97748badbd7bb676982db16/proto/qgb/types.proto#L31-L47) or [`Valset`](https://github.com/celestiaorg/celestia-app/blob/801a0d412631989ce97748badbd7bb676982db16/proto/qgb/types.proto#L17-L29).
```

Reply:
[7f28cb9](https://github.com/celestiaorg/celestia-app/pull/1052/commits/7f28cb9584fe4cd7eba0eaf2dc9f4b75757c87f0)